packages/twenty-server/src/engine/metadata-modules/field-metadata/interfaces/composite-type.interface.ts

```diff
@@ -0,0 +1,14 @@
+import { FieldMetadataType } from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
+
+export interface CompositeProperty {
+  name: string;
+  description?: string;
+  type: FieldMetadataType;
+  hidden: 'input' | 'output' | true | false;
+  isRequired: boolean;
```

Comment:
isNullable, isRequired is tied to the input only, which is a consequence of defaultValue + isNullable